bean-validation/2.0/_index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Jakarta Bean Validation Spec 2.0"
+date: 2019-08-02
+summary: "Initial Jakarta release of Jakarta Bean Validation"
+---
+Jakarta Bean Validation defines a metadata model and API for JavaBean and method validation.
+
+* [Jakarta Bean Validation 2.0 Specification Document](./bean-validation_2.0.pdf) (PDF)
+* [Jakarta Bean Validation 2.0 Specification Document](./bean-validation_2.0.html) (HTML)
+* [Jakarta Bean Validation 2.0 Javadoc](./apidocs)
+* [Jakarta Bean Validation 2.0 TCK](http://downloads.eclipse.org/jakarta/bean-validation/2.0.5/beanvalidation-tck-dist-2.0.5.zip)
+* Maven coordinates
+  * [jakarta.validation:jakarta.validation-api:jar:2.0.2](https://search.maven.org/artifact/jakarta.validation/jakarta.validation-api/2.0.2/jar)
```

Comment:
"2.0.2" -> "2.0.5"

Reply:
No, the API version is 2.0.2 indeed; 2.0.5 is the TCK version.

Reply:
Oops, you're right.